documentation/services_master_puppetserver.markdown

```diff
@@ -17,7 +17,7 @@ This page describes the generic requirements and run environment for Puppet Serv
 
 Puppet provides Puppet Server packages for Red Hat Enterprise Linux, RHEL-derived distros, Fedora, Debian, and Ubuntu.
 
-If we don't provide a package for your system, you can run Puppet Server from source on any POSIX server with JDK 1.7 or later. See [Running from Source](./dev_running_from_source.markdown) for more details.
+If we don't provide a package for your system, you can run Puppet Server from source on any x86_64 Linux server with JDK 1.8 or 11. See [Running from Source](./dev_running_from_source.markdown) for more details. Other POSIX compatible servers or versions of Java are unsupported by Puppet though may be possible. Please join our community Slack from help with non-supported operating systems, architectures, or JRE versions.
```

Comment:
suggest:

Other POSIX compatible servers or versions of Java are unsupported by Puppet, although they may work. Join our [community Slack](https://slack.puppet.com) for help with non-supported operating systems, architectures, or JRE versions.